tests/test_class_to_config.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function
+
+import environ
+
+
+@environ.config(prefix="APP")
+class AppConfig(object):
+    host = environ.var("127.0.0.1")
+    port = environ.var(5000, converter=int)
+
+
+def test_default():
```

Comment:
Please add [test docstrings](https://jml.io/pages/test-docstrings.html) to all tests. Believe me, you'll be glad for it, if you come back to the code in a year.

Reply:
Done. This i like on PRs to other's projects - it pays back by giving me a lesson on good coding style. Thanks.